recipes/filter-api-validator.rb

```diff
@@ -0,0 +1,35 @@
+#
+# Cookbook Name:: repose
+# Recipe:: filter-header-normalization
+#
+# Copyright (C) 2013 Rackspace Hosting
+#
+
+include_recipe 'repose::install'
+
+unless node['repose']['filters'].include? 'api-validator'
+  filters = node['repose']['filters'] + ['api-validator']
+  node.normal['repose']['filters'] = filters
+end
+
+template "#{node['repose']['config_directory']}/validator.cfg.xml" do
+  owner node['repose']['owner']
+  group node['repose']['group']
+  mode '0644'
+  variables(
+    :multi_role_match => node['repose']['api_validator']['multi_role_match'],
+    :delegating_quality => node['repose']['api_validator']['delegating_quality'],
+    :validators => node['repose']['api_validator']['validators']
+  )
+  notifies :restart, 'service[repose-valve]'
+end
+
+cookbook_file "allfeeds_observer.wadl" do
```

Comment:
These files probably need to be configurable since I imagine different products will want to bring their own wadls. We could do this by making the `cookbook` attribute of these `cookbook_file` resources configurable. Something like this maybe?

```ruby
node['repose']['api_validator']['wadl_filenames'].each do |f|
  cookbook_file f do
    cookbook node['repose']['api_validator']['wadl_cookbook']
    path "#{node['repose']['config_directory']}/#{f}.wadl"
    action :create_if_missing
  end
end
```